versioned_docs/version-8.7/components/modeler/web-modeler/web-modeler-settings.md

```diff
@@ -0,0 +1,70 @@
+---
+id: web-modeler-settings
+title: Web Modeler settings
+description: You can configure email notifications and process application deployment policies in the Web Modeler settings.
+---
+
+import WebModelerSettings from './img/web-modeler-settings/web-modeler-settings.png'
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+Navigate to the Web Modeler settings to configure email notifications and the process application deployment policy.
+You can find the settings by clicking on your user icon in the top right corner of the Web Modeler and selecting **Settings**.
+
+<img src={WebModelerSettings} alt="Web Modeler settings" />
+
+## Email notifications
+
+Configure the projects for which you will receive email notifications when a collaborator mentions you in a comment.
+
+## Process application deployment
+
+Organization admins can configure the deployment policy for process applications in the Web Modeler settings.
+
+<Tabs groupId="deployment-permissions" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+
+<TabItem value='saas'>
+
+By default, only organization administrators can deploy process applications to clusters marked as
+[production stages](/docs/components/modeler/web-modeler/process-application-pipeline.md#deployment-pipeline-stages) from Web Modeler.
+
+You can change this in the **process application deployment** settings.
+There, you can enable non-admin users with deployment permission to deploy process application versions to production stage clusters
+after a collaborator has reviewed and approved the process application version using the
+[process application review](/docs/components/modeler/web-modeler/process-application-pipeline.md#review) feature.
+This setting can only be configured by organization admins and applies to all process applications in the organization.
+
+</TabItem>
+
+<TabItem value='self-managed'>
+
+By default, only users with the **Web Modeler Admin** role can deploy process applications to
+clusters marked as [production stages](/docs/components/modeler/web-modeler/process-application-pipeline.md#deployment-pipeline-stages) from Web Modeler.
+
+You can change this in the **Process application deployment** settings.
+There, you can enable non-admin users with deployment permission to deploy process application versions to production stage clusters
```

Comment:
How do you enable this?

Reply:
It's a toggle. How should I update the text to reflect this?